### ISSUE TITLE
Setup a fake data analysis file, send it to pre-register it and verify that this is what we want to do.

### ISSUE THREAD

**Issue by jwbowers** [sbstusa/spilloverdesign#5]
It is probably worth setting up a fake data file so that we can write up a pre-analysis plan and getting clear on what we want to do with the data once it arrives.

**Comment by nahiggins** [comment]
Yeah, I agree.  I'll scramble some data and put it up so that we can use it to write our file.  Then we can put all of the code in one place.
